.github/workflows/ci.yml

```diff
@@ -259,8 +249,7 @@ jobs:
             ubuntu-16.04-shared-ccache-refs/heads/main-
       - id: install_deps
         run: |
-          echo "::set-env name=CMAKE_BUILD_OPTIONS::-DENABLE_PCAP=TRUE"
           sudo apt install -y ccache
-          echo "::set-env name=PATH::/usr/lib/ccache:$PATH"
+          echo "PATH=/usr/lib/ccache:$PATH" >> $GITHUB_ENV
       - id: build
         run: ./bootstrap.sh -e -t && cd build  && cmake -DUSE_SHARED_LIBS=ON -DSTRICT_GSL_CHECKS=AUDIT .. &&  cmake --build . --parallel 4  && make test ARGS="--timeout 300 -j4 --output-on-failure"
```

Comment:
It looks like you manually removed the docker timeout change from this cherry-picked commit.  Why?

It's OK if you have a good reason, but there is a danger that this change will be accidentally reverted when this branch is merged back to main.

Also, if you remove this part of the commit, please remove it from the commit message, too -- or, even better, mention in the commit message that you have removed this.